### PR TITLE
KS-SK1163-DDGR-KFT-Full-Update-DynHost-TRANSLATIONS

### PR DESCRIPTION
**Pull request by Kazarus4**
All translations are done.

Need proof for DE, EN, Latin and PL versions before merge.

### CODE REVIEW

**Comment by Kazarus4** [comment]
EN-GB to EN-IE duplications are done after EN-GB proof

**Review by bs-ovh — APPROVED** [review]

File: pages/web/domains/dns_dynhost/guide.es-es.md [hunk 1]
```diff
@@ -1,35 +1,56 @@
...
 
-**Última actualización: 05/09/2018**
+> [!primary]
+> Esta traducción ha sido generada de forma automática por nuestro partner SYSTRAN. En algunos casos puede contener términos imprecisos, como en las etiquetas de los botones o los detalles técnicos. En caso de duda, le recomendamos que consulte la versión inglesa o francesa de la guía. Si quiere ayudarnos a mejorar esta traducción, por favor, utilice el botón «Contribuir» de esta página.
+>
 
 ## Objetivo
 
-La zona DNS (Domain Name System) de un dominio es el archivo de configuración en el que se almacena su información técnica en forma de registros. Existen diversos motivos por los que puede ser necesaria la actualización dinámica de un registro DNS para evitar la interrupción prolongada de un servicio. Esto sucede, por ejemplo, cuando el usuario aloja su propio servidor de juego sin disponer de una dirección IP fija. 
+La zona **D**omain **N**ame **S**ystem (**DNS***) de un dominio constituye el fichero de configuración de este último. y se compone de información técnica llamada *enregistrement DNS*. La zona DNS es, en cierto modo, un centro de afilamiento.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_dynhost/guide.es-es.md]:
```suggestion
La zona **D**omain **N**ame **S**ystem (**DNS**) de un dominio constituye el fichero de configuración de este último. y se compone de información técnica llamada *enregistrement DNS*. La zona DNS es, en cierto modo, un centro de afilamiento. 
```

File: pages/web/domains/dns_dynhost/guide.es-es.md [hunk 2]
```diff
@@ -1,35 +1,56 @@
...
 
 - Tener acceso a la gestión del dominio desde el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es){.external}.
 - Utilizar la configuración de OVHcloud (es decir, sus servidores DNS) para el dominio en cuestión. 
-- El registro DynHost que vaya a crear no debe existir en la zona DNS de OVHcloud del dominio como registro A.
+- El registro DynHost que vaya a crear no debe existir en la zona DNS de OVHcloud del dominio como registro "A".
 
 > [!warning]
 >
-> - Si el dominio no utiliza los servidores DNS de OVHcloud, debe ponerse en contacto con el proveedor que gestione su configuración para que le informe de los pasos a seguir.
+> - Si el dominio no utiliza los servidores DNS de OVHcloud, contacte con el proveedor o proveedor que gestione su configuración DNS para conocer el procedimiento a seguir.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_dynhost/guide.es-es.md]:
```suggestion
> - Si el dominio no utiliza los servidores DNS de OVHcloud, contacte con el proveedor que gestione su configuración DNS para conocer el procedimiento a seguir.
```

File: pages/web/domains/dns_dynhost/guide.it-it.md [hunk 3]
```diff
@@ -50,33 +75,54 @@ Una volta che tutto è pronto per creare il record, torna alla pagina principale
 |Campo|Descrizione|
 |---|---|
 |Sottodominio|Inserisci il nome del sottodominio a cui è associato il record DNS da aggiornare dinamicamente. Questo sottodominio deve coincidere con quello inserito durante la creazione dell’utente DynHost.|
-|IP di destinazione|Inserisci l’indirizzo IP che deve essere utilizzato dal record DNS. Secondo il funzionamento del DynHost, l’IP verrà aggiornato in un secondo momento.|
+|IP di destinazione|Inserisci l'indirizzo IP (solo IPv4) che deve essere utilizzato dal record DNS. È generalmente l'indirizzo IP pubblico del tuo *box* Internet o del tuo server auto-ospitato. In base al principio del DynHost, verrà aggiornato automaticamente in seguito.|
+
+> [!primary]
+>
+> Per l'installazione di un DynHost è possibile utilizzare solo una **IPv4**. I **IPv6** non sono disponibili
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_dynhost/guide.it-it.md]:
```suggestion
> Per l'installazione di un DynHost è possibile utilizzare solo una **IPv4**. I **IPv6** non sono disponibili.
```

**Pull request merged**